salt/grains/core.py

```diff
@@ -1482,7 +1482,7 @@ def os_data():
     grains.update(_ps(grains))
 
     # Load additional OS family grains
-    if grains['os_family'] == "RedHat":
+    if grains['os_family'] in ['RedHat', 'SUSE']:
         grains['osmajorrelease'] = grains['osrelease'].split('.', 1)[0]
```

Comment:
No need to do that, since 'osrelease_info' is already been split. That said, it is just sufficient to:
```python
grains['osmajorrelease'] = grains['osrelease_info'][0]
```